### ISSUE TITLE
is jolokia 1.5 supported on Tomcat 8.5?

### ISSUE THREAD

**Issue by Iulian-Mitrea** [rhuss/jolokia#379]
After I deploy jolokia 1.5.0 on Tomcat 8.5 env I get the following message:
"_Environment Configuration Probems: It seems Jolokia agent was not installed on test environment or missconfigured. Error thrown: 
[testng] java.lang.NullPointerException_"

I used the same steps on Tomcat 8.0 and everything worked fine.
If I access ip:port/jolokia I get the following message:
"_info":{"product":"tomcat","vendor":"Apache","version":"8.5.23"}},"timestamp":1523629269,"status":200}_"

**Comment by rhuss** [comment]
If you could tell us the steps how you deployed Jolokia, and provide please a bit more of context, I could have a look at it. Tomcat 8.5 should be (and is) supported for sure and has been reported to run successfully with Jolokia

Thanks!

**Comment by Iulian-Mitrea** [comment]
Hi @rhuss ,
In my case I deploy jolokia.war into a Tomcat 8.5 env.
After deployment I access http://ip:port/jolokia and I get the confirmation message that everything works fine. Next, I run a set of automation verification and here I got the error messages saying that jolokia is misconfigured. 
I did some investigation and actually the issue was on my side. I was typing a wrong port name for jolokia in the automation test properties. I fixed this and now everything works fine.

Thank you!
